src/game/Spells/SpellEffects.cpp

```diff
@@ -4066,7 +4066,8 @@ void Spell::EffectTaunt(SpellEffectIndex eff_idx)
     // Also use this effect to set the taunter's threat to the taunted creature's highest value
     if (unitTarget->CanHaveThreatList() && unitTarget->GetThreatManager().getCurrentVictim())
     {
-        unitTarget->GetThreatManager().addThreat(m_casterUnit, unitTarget->GetThreatManager().getCurrentVictim()->getThreat());
+        //Add the difference of the Current highest value threat and the taunter's threat
```

Comment:
Please add an extra space between `//` and `Add` and remove uppercase from `Current`. :pray:

Reply:
Made a second commit to update, hope that's alright.